index.html

```diff
@@ -781,9 +781,17 @@ <h2>Newsletter</h2>
 
         <div class="section-header">
           <h2>Registre-se</h2>
-          <p>Em breve...</p>
+
+          <p>A inscrição não é obrigatória para participar, mas vai nos ajudar a fazer um evento
+            melhor preparado para vocês.</p>
+
+          <p>Precisamos também de pessoas disponíveis a auxiliar quem tiver menos experiência.<br>
+            Favor, preencha o formulário informando se puder ficar acessível para ajudar com código,

```

Comment:
Acho que poderia ser:

```suggestion
            Favor, preencha o formulário informando se puder ficar acessível para ajudar com o desenvolvimento de códigos,
```